eerie-sys/build.rs

```diff
@@ -159,6 +159,7 @@ fn main() {
                 ("IREE_BUILD_SAMPLES", "OFF"),
                 ("IREE_BUILD_BINDINGS_TFLITE", "OFF"),
                 ("IREE_BUILD_BINDINGS_TFLITE_JAVA", "OFF"),
+                ("CMAKE_BUILD_TYPE", "RelWithDebInfo"),
```

Comment:
https://docs.rs/cmake/latest/cmake/struct.Config.html#method.profile
[CMake-rs](https://crates.io/crates/cmake) automatically sets the cmake build profile according to the rust build flags. Have you encountered any occasions where this didn't work?

Reply:
I didn't know about this, I just saw debug builds and naively assumed it would always be that. My bad, I'll remove it again.